client/lib/trigram/test/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
```

Comment:
Perhaps we could use `jest`'s expect instead?

Reply:
Done